src/modules/project/sagas.ts

```diff
@@ -205,13 +204,11 @@ export function* projectSaga(builder: BuilderAPI) {
         onProgress: progress => store.dispatch(setExportProgress(progress))
       })
     )
-
     for (const filename of Object.keys(files)) {
       zip.file(filename, files[filename])
     }
 
-    const artifact = yield call(zip.generateAsync, { type: 'blob' })
-    saveAs(artifact, `${sanitizedName}.zip`)
+    yield call(download, zip, sanitizedName)
```

Comment:
This was the line that was breaking the export. I have no idea why it suddenly stopped working.. and I dunno why if I wrap it in an async function it works again, but it does work now 🤷‍♂️ .

Reply:
Just random guessing, but it might be because of something like this https://redux-saga.js.org/docs/api/#callcontext-fn-args

Reply:
maybe the generateAsync is not being provided the correct `this` which might be fixed by using the call alternatives

Reply:
good catch -- `this` was `null` and causing the function to blow up